php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCompletionItem.java

```diff
@@ -359,12 +368,99 @@ public String getInsertPrefix() {
                     assert false : "[" + tpl + "] should start with [" + extraPrefix + "]";
                 }
             }
+            if (getCodeCompletionType() == CodeCompletionType.AUTO_IMPORT) {
+                String fqName = ifq.getFullyQualifiedName().toString().substring(CodeUtils.NS_SEPARATOR.length());
+                String name = getName();
+                String useType = getUseType();
+                String aliasName = CodeUtils.EMPTY_STRING;
```

Comment:
This is not used at the moment. However, it would be useful if someone requests importing as an alias(when the use list has the same name) in the future.

Reply:
I see.